client/go/commands/flags_test.go

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright (c) 2015 The heketi Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package commands
+
+import (
+	"bytes"
+	"github.com/gorilla/mux"
+	"github.com/heketi/heketi/apps/glusterfs"
+	"github.com/heketi/heketi/tests"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestJsonFlagsCreate(t *testing.T) {
```

Comment:
Should add a test that shows that not having -json should print NO JSON output.  In other words.. unmarshalling the output would return an error.